packages/@jsii/spec/lib/configuration.ts

```diff
@@ -77,16 +77,6 @@ export interface Config {
        * version field
        */
       versionSuffix?: string;
-
-      /**
-       * whether the assembly should be strong-name signed
-       */
```

Comment:
Is there a chance this is going to somehow break existing customers that pass these values?

Reply:
Impossible - this feature **could not possibly** have been used (it never worked!!!)